<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 232, 232);
</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sprzedawca</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Nabywca</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="buyersEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Imie Nazwisko/ Nazwa Firmy                                    Ulica                                                                          Kod Pocztowy, Miasto                                                          NIP</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sellersEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>231</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Artifakt Element'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>Imie Nazwisko/ Nazwa Firmy                                    Ulica                                                                          Kod Pocztowy, Miasto                                                          NIP</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Data sprzedaży</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Data wystawienia</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Termin płatności</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="paymentFormEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Przelew/Gotówka</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Forma płatności</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Uwagi</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="notesEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>(Np. numer konta)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton0">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>23%</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Podatek</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5%</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>zw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>505</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(73, 84, 100);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string>Cofnij</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Klauzula pod fakturą</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="notesBottomEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>(Np. zwolnienie z podatku VAT)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sposób dodawania</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="multipleClientBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Każdy klient osobno</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="singleClientBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wszyscy klienci pod 
jedną nazwą produktu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nr Faktury</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="invoiceNumberEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>01/01/2022</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>505</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>WhatsThisCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zapisywanie do pliku automatycznie wczytujące dane po zamknięciu aplikacji.&lt;/p&gt;&lt;p&gt;W przypadku jednorazowych danych nie ma konieczności zapisu - dane do aktualnej faktury wczytują się również bez zapisu.&lt;br/&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(73, 84, 100);
border-color: rgb(0, 0, 0);
border: 2px solid;
border-radius: 15px;
color: rgb(232, 232, 232);</string>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nazwa produktu</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="productNameEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Usługa pośrednictwa finansowego</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Artifakt Element</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
